src/repositories/userRepository.ts

```diff
@@ -14,14 +14,14 @@ export const createUser = async (
   username: string,
   password: string,
   role: 'customer' | 'admin' = 'customer'
```

Comment:
extend this to an enum like:

export enum Role {
  Customer = 'customer',
  Admin = 'admin'
}
Meaning you can then have
```suggestion
  role: Role = Role.Customer
```


Ensure you run the code before pushing it